src/EditorFeatures/CSharpTest/ConvertToRawString/ConvertRegularStringToRawString_FixAllTests.cs

```diff
@@ -0,0 +1,966 @@
+                    <Document>
+            class Program3
+            {
+                void M3()
+                {
+                    var singleLine1 = "a";
+                    var singleLine2 = @"goo""bar";
+                }
+            }
+                    </Document>
+                </Project>
+            </Workspace>
+            """,
+            """"
+            <Workspace>
+                <Project Language="C#" AssemblyName="Assembly1" CommonReferences="true">
+                    <Document>
+            partial class Program1
+            {
+                void M1()
+                {
+                    var singleLine1 = """a""";
+                    var singleLine2 = """goo"bar""";
+                }
+
+                void M2()
+                {
+                    var singleLine1 = """a""";
+                    var singleLine2 = """goo"bar""";
+                }
+            }
+                    </Document>
+                    <Document>
+            partial class Program1
+            {
+                void M3()
+                {
+                    var singleLine1 = """a""";
+                    var singleLine2 = """goo"bar""";
+                }
+            }
+
+            class Program2
+            {
+                void M2()
+                {
+                    var singleLine1 = "a";
+                    var singleLine2 = @"goo""bar";
+                }
+            }
+                    </Document>
+                </Project>
+                <Project Language="C#" AssemblyName="Assembly2" CommonReferences="true">
+                    <Document>
+            class Program3
+            {
+                void M3()
+                {
+                    var singleLine1 = "a";
+                    var singleLine2 = @"goo""bar";
+                }
+            }
+                    </Document>
+                </Project>
+            </Workspace>
+            """");
+        }
+
+        [Fact]
+        public async Task FixAllCommonRoslynTestPattern1()
+        {
+            await TestInRegularAndScriptAsync(
+                """
+                class C
+                {
+                    void M()
+                    {
+                        await TestAsync(
+                {|FixAllInDocument:|}@"class X
+                {
+                }",
+                @"class Y
+                {
+                }");
+                    }
+                }
+                """,
+                """"
+                class C
+                {
+                    void M()
+                    {
+                        await TestAsync(
+                            """
+                            class X
+                            {
+                            }
+                            """,
+                            """
+                            class Y
+                            {
+                            }
+                            """);
```

Comment:
this output is much better than before.  both arguments are now properly indented, while the contents of that argument stay appropriately non-indented.